bitnami/keycloak/templates/servicemonitor.yaml

```diff
@@ -21,7 +21,7 @@ spec:
   jobLabel: {{ .Values.metrics.serviceMonitor.jobLabel | quote }}
   endpoints:
     - port: tcp-metrics
-      path: "/metrics"
+      path: "{{ trimSuffix "/" .Values.httpRelativePath }}/metrics"
```

Comment:
I like this solution! Could you please apply the same approach to this line to keep the consitency in the chart?
https://github.com/bitnami/charts/blob/436a967c4c2b5d954d149e3a934ea6c5b82585c7/bitnami/keycloak/templates/statefulset.yaml#L221